styles/styles.css

```diff
@@ -0,0 +1,180 @@
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+img {
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.center-div {
+  margin: 0 auto;
+}
+
+.container {
+  width: 1440px;
+}
+
+.container-small {
+  width: 950px;
+}
+
+.container-footer {
+  width: 950px;
+}
+
+.flex {
+  display: flex;
+}
+
+.underline:hover {
+  text-decoration: underline;
+}
+
+.lighter {
+  font-weight: lighter;
+}
+
+.nav-radius-right {
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+.lucida {
+  font-family: Roboto, Helvetica, Arial, Verdana, sans-serif;
+}
+
+.c-white {
+  color: white;
+}
+
+.title1 {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 70px;
+  line-height: 90px;
+  font-weight: 100;
+}
+
+.title2 {
+  font-size: 33px;
+  line-height: 40px;
+  width: 60%;
+  text-align: center;
+}
+
+.title3 {
+  font-size: 25px;
+  line-height: 60px;
+}
+
+ul li {
+  display: inline;
+}
+
+nav ul li {
+  width: 106px;
+}
```

Comment:
This is ok. But in larger projects, this could affect elements that not necessarily the programmer wanted to change. Preferring specific classes instead fo the more general ones helps with this issue. Example:

```
ul li {
  display: inline;
}

nav ul li {
  width: 106px;
}
```

**Improved** :+1: 
```
.d-inline {
  display: inline; 
}

nav__item {
  width: 106px;
}
```